src/main/java/com/mindee/utils/geometry/BoundingBoxUtils.java

```diff
@@ -0,0 +1,28 @@
+package com.mindee.utils.geometry;
+
+import java.util.Arrays;
+import java.util.DoubleSummaryStatistics;
+
+import com.mindee.model.geometry.Point;
+import com.mindee.model.geometry.Polygon;
+
+public final class BoundingBoxUtils {
+  private BoundingBoxUtils() {
+  }
+
+  public static Polygon createFrom(Polygon polygon) {
```

Comment:
Minor - since the return type is a Polygon, the method name createFrom does not fully explain we are returning a bounding box. Consider renaming "createFrom" to something more descriptive like "createBoundingBoxFrom" ?